ndk-patches/27/stdlib.h.patch

```diff
@@ -1,22 +1,20 @@
 --- ./usr/include/stdlib.h.orig	2021-08-16 11:50:47.432239182 +0200
 +++ ./usr/include/stdlib.h	2021-08-16 11:51:31.102222894 +0200
-@@ -33,6 +33,7 @@
+@@ -33,6 +33,9 @@
  #include <bits/wait.h>
  #include <malloc.h>
  #include <stddef.h>
++#if !defined(__swift__)
 +#include <stdint.h>
++endif
```

Comment:
Missing `#` here, `#endif`

Reply:
Thanks

Reply:
@licy183, I didn't pull again before fixing this typo just now. Feel free to push your local branch again with my last commit added if I missed something.